lib/webdrivers/chromedriver.rb

```diff
@@ -31,18 +35,57 @@ def downloads
         Webdrivers.logger.debug "Versions previously located on downloads site: #{@downloads.keys}" if @downloads
 
         @downloads ||= begin
-          doc = Nokogiri::XML.parse(get(base_url))
+          doc   = Nokogiri::XML.parse(get(base_url))
           items = doc.css("Contents Key").collect(&:text)
-          items.select! {|item| item.include?(platform)}
+          items.select! { |item| item.include?(platform) }
           ds = items.each_with_object({}) do |item, hash|
-            key = normalize item[/^[^\/]+/]
+            key       = normalize item[/^[^\/]+/]
             hash[key] = "#{base_url}/#{item}"
           end
           Webdrivers.logger.debug "Versions now located on downloads site: #{ds.keys}"
           ds
         end
       end
 
+      # Returns release version from the currently installed Chrome version
+      #
+      # @example
+      #   73.0.3683.75 -> 73.0.3683
+      def release_version
+        chrome_version.scan(/\d+\.\d+\.\d+/)[0]
+      end
+
+      # Returns currently installed Chrome version
+      def chrome_version
+        ver = case platform
+                when 'win'
+                  chrome_on_windows
+                when /linux/
+                  chrome_on_linux
+                when 'mac'
+                  chrome_on_mac
+                else
+                  raise NotImplementedError, 'Your OS is not supported by webdrivers gem.'
+              end.chomp
+
+        # Google Chrome 73.0.3683.75 -> 73.0.3683.75
+        ver.scan(/\d+\.\d+\.\d+\.\d+/)[0]
+      end
+
+      def chrome_on_windows
+        reg = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe'
+        ps  = "(Get-Item (Get-ItemProperty '#{reg}').'(Default)').VersionInfo.ProductVersion"
+        `powershell #{ps}`
+      end
+
+      def chrome_on_linux
+        `google-chrome --version`
+      end
+
+      def chrome_on_mac
+        loc = Shellwords.escape '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
```

Comment:
Oh cool, I've learned about a new useful class, thanks.

Reply:
Yea, pretty useful. I found it in Sean Linsley's gist here: https://gist.github.com/seanlinsley/36a204862f08942b0667e941331f3a75